test/extensions/filters/network/common/fuzz/BUILD

```diff
@@ -59,3 +68,38 @@ envoy_cc_fuzz_test(
         "//test/config:utility_lib",
     ] + envoy_all_network_filters(),
 )
+
+envoy_cc_test_library(
+    name = "uber_writefilter_lib",
+    srcs = [
+        "uber_per_writefilter.cc",
+        "uber_writefilter.cc",
+    ],
+    hdrs = ["uber_writefilter.h"],
+    deps = [
+        ":network_writefilter_fuzz_proto_cc_proto",
+        "//source/common/config:utility_lib",
+        "//source/extensions/filters/network:well_known_names",
+        "//source/extensions/filters/network/common:utility_lib",
+        "//test/extensions/filters/network/common/fuzz/utils:network_filter_fuzzer_fakes_lib",
+        "//test/fuzz:utility_lib",
+        "//test/mocks/network:network_mocks",
+    ],
+)
+
+envoy_cc_fuzz_test(
+    name = "network_writefilter_fuzz_test",
+    srcs = ["network_writefilter_fuzz_test.cc"],
+    corpus = "network_writefilter_corpus",
+    # All Envoy network filters must be linked to the test in order for the fuzzer to pick
```

Comment:
Are the deps below added instead of `envoy_all_network_filters` to exclude postgres?

Reply:
> Are the deps below added instead of `envoy_all_network_filters` to exclude postgres?

`envoy_all_network_filters` includes 20 filters and only 5 of them are WriteFilters. And yes, I exclude postgres here. But another reason is to exclude all those ReadFilters to reduce linking time.

Reply:
Sounds good for now -- I just wonder if somoene adds a write filter it may get missed.